pages/views/examples/toggleable.njk

```diff
@@ -6,119 +6,119 @@ name: Toggleable
 tags: examples
 icon: examples/toggleable.svg
 aside:
-  components:
-    - esl-toggleable
-    - esl-trigger
-  examples:
-    - accordion
-    - tabs
+components:
+- esl-toggleable
+- esl-trigger
+examples:
+- accordion
+- tabs
```

Comment:
That's will not work.
Frontmetter it is yml syntax, so each indent matters.
So it is not right configuration here right now